packages/scandipwa/src/component/Menu/Menu.config.js

```diff
@@ -12,3 +12,4 @@
  */
 
 export const SCROLL_DEBOUNCE_DELAY = 10;
+export const SUBCATEGORY_URL = '/menu/subcategory';
```

Comment:
I think this could have been easily done without introducing the new URL, by manipulating the history state, but OK